### DISCUSSION TITLE
An option to provide indentation in the JSON file.

### DISCUSSION THREAD

**Discussion by WiseDevHelper** [Androz2091/discord-giveaways#383] [Ideas]
**Is your feature request related to a problem? Please describe.**  

Whenever I try to edit the giveaway file (or it gets corrupted), it is really tedious because all the giveaways are in a single line.

**Describe the solution you'd like**

An option to provide indentation in the JSON file.

**Describe alternatives you've considered**  

Implement this manually.

**Comment by WiseDevHelper** [comment]
@Nico105 Thoughts?

**Comment by twlite** [comment]
that would increase your JSON file size. You can manually format your json with `JSON.stringify(data, null, "\t")`

**Comment by WiseDevHelper** [comment]
> that would increase your JSON file size. You can manually format your json with `JSON.stringify(data, null, "\t")`

Yes, that is why I suggested to add it as an option.

**Comment by Shashank3736** [comment]
There are many ways to do this without a new option.
Like vscode has an extension to prettify JSON file. You can use that. Having an extra option for that is not really a requirement. As package recommend the usage of other methods to edit your giveaway. Your custom changes may cause some data troubles.
